src/sdk/TranslationRecognizer.ts

```diff
@@ -206,7 +206,7 @@ export class TranslationRecognizer extends Recognizer {
      * @param cb - Callback that received the translation has stopped.
      * @param err - Callback invoked in case of an error.
      */
-    public stopContinuousRecognitionAsync(cb?: () => void, err?: (e: string) => void): void {
+    public stopContinuousRecognitionAsync(cb?: (v: unknown) => void, err?: (e: string) => void): void {
```

Comment:
>v: unknown [](start = 48, length = 10)

Why does the stop callback now take a  parameter?

Reply:
this is called on line 276 of ConversationTranslator.stopTranscribingAsync(). VS Code was complaining about:
```
                await new Promise((resolve: () => void, reject: (error: string) => void): void => {
                    this.privTranslationRecognizer?.stopContinuousRecognitionAsync(resolve, reject);
                });
```
It wanted resolve to have a "value: unknown" param, and this TranslationRecognizer change is downstream of that.

Reply:
reverted